pkg/cmd/edge_applications/publish/publish.go

```diff
@@ -449,15 +495,17 @@ func (cmd *PublishCmd) createApplication(client *apiapp.Client, ctx context.Cont
 	reqApp.SetDeliveryProtocol("http,https")
 	application, err := client.Create(ctx, &reqApp)
 	if err != nil {
+		logger.Error("Create return error", zap.Error(err))
 		return 0, 0, fmt.Errorf(msg.ErrorCreateApplication.Error(), err)
 	}
-	fmt.Fprintf(cmd.F.IOStreams.Out, msg.EdgeApplicationsPublishOutputEdgeApplicationCreate, application.GetName(), application.GetId())
+	logger.FInfo(cmd.F.IOStreams.Out, fmt.Sprintf(msg.EdgeApplicationsPublishOutputEdgeApplicationCreate, application.GetName(), application.GetId()))
 	reqUpApp := apiapp.UpdateRequest{}
 	reqUpApp.SetEdgeFunctions(true)
 	reqUpApp.SetApplicationAcceleration(true)
 	reqUpApp.Id = application.GetId()
 	application, err = client.Update(ctx, &reqUpApp)
 	if err != nil {
+		logger.Error("Update return error", zap.Error(err))
```

Comment:
these Logs do nothing... because the line below also returns the Err and it will be printed
If you execute this, you will see the same error twice

Reply:
Não, pq uma coisa é eu ter o log do erro a outra é retornar o erro pra ser repassado